README.md

```diff
@@ -0,0 +1,78 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo-List
+
+> In this repository, we build a simple and todo list app. Look at the minimalist version we tried to build below:
```

Comment:
- [ ] In order to keep your code clean, it would be better to remove all code comments that don't describe the code itself